### PR TITLE
Update Dockerfiles to io.js v1.6.0

### PR DESCRIPTION
**Pull request by Starefossen**
~~BLOCKED BY iojs/io.js#1157~~

This PR updates Dockerfiles to v1.6.0 of io.js.

### CODE REVIEW

File: 1.6/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM buildpack-deps:jessie
 RUN gpg --keyserver pool.sks-keyservers.net --recv-keys 9554F04D7259F04124DE6B476D5A82AC7E37093B DD8F2338BAE7501E3DD5AC78C273792F7D83545D
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV IOJS_VERSION 1.5.1
+ENV IOJS_VERSION 1.6.1
```
Comment (arthurschreiber) [inline 1.6/Dockerfile]: Shouldn't that be `1.6.0` instead?

**Comment by pesho** [comment]
`aliases` in `generate-stackbrew-library.sh` needs to be updated as well.

File: 1.6/Dockerfile (hunk 1 above)
Comment (Starefossen) [inline 1.6/Dockerfile]: Fixed

**Comment by retrohacker** [comment]
:heart:

**Comment by Starefossen** [comment]
I guess this should be good to go now 😄

**Comment by Starefossen** [comment]
Maybe we should hold off merging this until iojs/io.js#1208 has been fixed and v1.61 is released.

**Comment by pesho** [comment]
It has been [confirmed](https://github.com/iojs/io.js/issues/1208#issuecomment-83788463) that 1.6.1 will follow soon. Holding off for now.

**Comment by pesho** [comment]
@Starefossen can you update this PR for 1.6.1 in the meantime?

**Comment by pesho** [comment]
Merging, will update to 1.6.1 in a separate PR.

**Pull request merged**